eth/ethconfig/config.go

```diff
@@ -69,7 +69,7 @@ var Defaults = Config{
 	GasPrice:           big.NewInt(0.25 * params.Shannon),
 
 	TxPool:      core.DefaultTxPoolConfig,
-	RPCGasCap:   25000000,
+	RPCGasCap:   150000000,
```

Comment:
150000000 is too big. This value is 50000000 in geth.

Reply:
might be. thats why I'm asking to add a special flag to give other node providers possibility to configure it by themselves

Reply:
Yes, flag is a good idea. Please search `RPCGasCap` and `RPCGlobalGasCapFlag` in geth.